docs/src/user_docs/stdlib/collections.md

```diff
@@ -0,0 +1,22 @@
+# Collections
+Namespace `std::crypto` contains modules for commonly authenticated data structures.
```

Comment:
typo? Should this be `std::collections` instead of `std::crypto`? Also, the phrase "commonly authenticated data structures" feels a bit odd